docs/tutorials/custom_experiment.rst

```diff
@@ -0,0 +1,150 @@
+Writing a custom experiment
+===========================
+
+Qiskit-Experiments is designed to be easily customizable. To create an experiment subclass
+based on either the :class:`.BaseExperiment` class or an existing experiment, you should:
+
+- Implement the abstract :meth:`.BaseExperiment.circuits` method.
+  This should return a list of ``QuantumCircuit`` objects defining
+  the experiment payload.
+
+- Call the :meth:`BaseExperiment.__init__` method during the subclass
+  constructor with a list of physical qubits. The length of this list must
+  be equal to the number of qubits in each circuit and is used to map these
+  circuits to this layout during execution.
+  Arguments in the constructor can be overridden so that a subclass can
+  be initialized with some experiment configuration.
+
+Optionally the following methods can also be overridden in the subclass to
+allow configuring various experiment and execution options
+
+- :meth:`.BaseExperiment._default_experiment_options`
+  to set default values for configurable option parameters for the experiment.
+
+- :meth:`.BaseExperiment._default_transpile_options`
+  to set custom default values for the ``qiskit.transpile`` used to
+  transpile the generated circuits before execution.
+
+- :meth:`.BaseExperiment._default_run_options`
+  to set default backend options for running the transpiled circuits on a backend.
+
+- :meth:`.BaseExperiment._default_analysis_options`
+  to set default values for configurable options for the experiments analysis class.
+  Note that these should generally be set by overriding the :class:`.BaseAnalysis`
+  method :meth:`.BaseAnalysis._default_options` instead of this method except in the
+  case where the experiment requires different defaults to the used analysis class.
```

Comment:
This last sentence is too advanced for a beginner. I suggest to move it to the method's doc string, and explain about it in the examples below.

Reply:
This was outdated anyways. I've updated the paragraph.